### PR TITLE
Multi tenant mongo

### PR DESCRIPTION
**Pull request by fhuaulme**
**Please check if the PR fulfills these requirements**

- [x] The commit(s) message(s) follows our [guidelines](https://github.com/talend/tools/blob/master/tools-root-github/CONTRIBUTING.md#commit-message-format) ?
- [x] Tests for the changes have been added (for bug fixes / features) ?
- [x] Docs have been added / updated (for bug fixes / features) ?
- [x] The code coverage on new code >75%
- [x] The new code does not introduce new technical issues (sonar / eslint)

### CODE REVIEW

**Comment by build-talend-common** [comment]
Pull request test result: [success](https://ci-common.datapwn.com/job/daikon_pull-request_master/216/) [:star:](https://ci-common.datapwn.com/job/daikon_pull-request_master/216/)

**Comment by build-talend-common** [comment]
Pull request test result: [success](https://ci-common.datapwn.com/job/daikon_pull-request_master/218/) [:star:](https://ci-common.datapwn.com/job/daikon_pull-request_master/218/)

**Comment by build-talend-common** [comment]
Pull request test result: [failure](https://ci-common.datapwn.com/job/daikon_pull-request_master/220/) [:x:](https://ci-common.datapwn.com/job/daikon_pull-request_master/220/)

**Comment by build-talend-common** [comment]
Pull request test result: [success](https://ci-common.datapwn.com/job/daikon_pull-request_master/234/) [:star:](https://ci-common.datapwn.com/job/daikon_pull-request_master/234/)

**Comment by build-talend-common** [comment]
Pull request test result: [success](https://ci-common.datapwn.com/job/daikon_pull-request_master/235/) [:star:](https://ci-common.datapwn.com/job/daikon_pull-request_master/235/)

**Comment by build-talend-common** [comment]
Pull request test result: [success](https://ci-common.datapwn.com/job/daikon_pull-request_master/254/) [:star:](https://ci-common.datapwn.com/job/daikon_pull-request_master/254/)

**Comment by build-talend-common** [comment]
Pull request test result: [success](https://ci-common.datapwn.com/job/daikon_pull-request_master/255/) [:star:](https://ci-common.datapwn.com/job/daikon_pull-request_master/255/)

**Comment by build-talend-common** [comment]
Pull request test result: [success](https://ci-common.datapwn.com/job/daikon_pull-request_master/259/) [:star:](https://ci-common.datapwn.com/job/daikon_pull-request_master/259/)

**Comment by build-talend-common** [comment]
Pull request test result: [success](https://ci-common.datapwn.com/job/daikon_pull-request_master/262/) [:star:](https://ci-common.datapwn.com/job/daikon_pull-request_master/262/)

**Comment by build-talend-common** [comment]
Pull request test result: [success](https://ci-common.datapwn.com/job/daikon_pull-request_master/263/) [:star:](https://ci-common.datapwn.com/job/daikon_pull-request_master/263/)

File: daikon-spring/daikon-spring-mongo/pom.xml [hunk 1]
```diff
@@ -0,0 +1,71 @@
...
+        <java.version>1.8</java.version>
+        <maven.compiler.source>1.8</maven.compiler.source>
+        <maven.compiler.target>1.8</maven.compiler.target>
+        <project.build.sourceEncoding>UTF-8</project.build.sourceEncoding>
+    </properties>
+    <dependencies>
+        <dependency>
+            <groupId>org.springframework</groupId>
+            <artifactId>spring-core</artifactId>
+            <version>4.3.4.RELEASE</version>
```
Comment (talend-vlesquere) [inline daikon-spring/daikon-spring-mongo/pom.xml]: what about the creation of a property for spring.version, i would use java.version wherever the 1.8 is used too.

**Comment by build-talend-common** [comment]
Pull request test result: [success](https://ci-common.datapwn.com/job/daikon_pull-request_master/280/) [:star:](https://ci-common.datapwn.com/job/daikon_pull-request_master/280/)

**Pull request merged**